### PR TITLE
Fixed #12437 -- Added css_classes to Form._html_output().

### PR DESCRIPTION
**Pull request by alimony**
See ticket: https://code.djangoproject.com/ticket/12437

### CODE REVIEW

File: django/forms/forms.py [hunk 1]
```diff
@@ -249,7 +250,8 @@ def _html_output(self, normal_row, error_row, row_ender, help_text_html, errors_
                         'label': '',
                         'field': '',
                         'help_text': '',
-                        'html_class_attr': html_class_attr,
```
Comment (MoritzS) [inline django/forms/forms.py]: Why did you make the `html_class_attr` field empty here?

Comment (alimony) [inline django/forms/forms.py]: I saw no real reason for it when creating an empty row for the sole purpose to insert other output. However, I digged through the tickets I could find on this (3512, 3686, 3515), and looked at the [original commit](https://github.com/django/django/commit/92803205cbcaaee16ac0eb724c45019a9d896aac) adding it, and it seems like it was written like that on purpose, although I find no reference to why. So I just pushed a new commit that reverts it, doesn't make a difference for this new patch :)

**Comment by timgraham** [comment]
buildbot, add to whitelist.

File: tests/forms_tests/tests/test_forms.py [hunk 2]
```diff
@@ -2274,11 +2274,50 @@ class SomeForm(Form):
...
+                    help_text_html=' %s',
+                    errors_on_separate_row=True,
+                )
+
+        form = SomeForm()
+        self.assertHTMLEqual(form.as_p(), '<p class=""></p>')
+
+    def test_field_with_css_class(self):
+        class SomeForm(Form):
+            some_field = CharField(widget=TextInput(attrs={'class': 'first_name_id'}))
```
Comment (timgraham) [inline tests/forms_tests/tests/test_forms.py]: is `widget=...` important for the test?

**Comment by alimony** [comment]
I think the widget was just a leftover from my dev process, I've removed it now and squashed some of the commits. Should I ideally squash everything into one commit?

**Comment by timgraham** [comment]
Yes, please. You can also use the commit message from the title of the pull request (see the [patch review checklist](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/submitting-patches/#patch-review-checklist) for the commit message guidelines).

**Comment by alimony** [comment]
I've squashed everything into one commit with a better message, will pay more attention to this in the future.

**Comment by timgraham** [comment]
merged in 1884bf8e8e9fb3f9f44a9ec216aca96f63974268, thanks!

**Pull request closed without merging**